quadriga/src/main/java/edu/asu/diging/quadriga/web/GraphUtil.java

```diff
@@ -0,0 +1,78 @@
+package edu.asu.diging.quadriga.web;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.bson.types.ObjectId;
+
+import edu.asu.diging.quadriga.core.model.DefaultMapping;
+import edu.asu.diging.quadriga.core.model.TripleElement;
+import edu.asu.diging.quadriga.web.model.GraphData;
+import edu.asu.diging.quadriga.web.model.GraphEdgeData;
+import edu.asu.diging.quadriga.web.model.GraphElement;
+import edu.asu.diging.quadriga.web.model.GraphElements;
+import edu.asu.diging.quadriga.web.model.GraphNodeData;
+import edu.asu.diging.quadriga.web.model.GraphNodeType;
+
+public class GraphUtil {
+
+    public static GraphElements mapToGraph(List<DefaultMapping> triples) {
```

Comment:
it looks like this create a graph of disconnected triples? if two nodes represent the same concept, it should be the same node.
also there needs to be a lot of javadoc for this class. And it should probably be in the service package and have an interface.